source/header.c

```diff
@@ -474,6 +472,10 @@ static void init_aws_byte_buf_raw(struct aws_byte_buf *buf) {
 int aws_cryptosdk_hdr_write(
     const struct aws_cryptosdk_hdr *hdr, size_t *bytes_written, uint8_t *outbuf, size_t outlen) {
     AWS_PRECONDITION(aws_cryptosdk_hdr_is_valid(hdr));
+    AWS_PRECONDITION(hdr->iv.len <= 255);  // uint8_t max value
```

Comment:
Why don't you use
```c
AWS_PRECONDITION(hdr->iv.len <= UINT8_MAX);
```
See [documentation](https://www.qnx.com/developers/docs/6.4.1/dinkum_en/c99/stdint.html#UINT8_MAX).

Reply:
Thanks, updated!